ledger-service/http-json-perf/BUILD.bazel

```diff
@@ -30,27 +30,17 @@ perf_runtime_deps = {
     da_scala_library(
         name = "http-json-perf-{}".format(edition),
         srcs = glob(["src/main/scala/**/*.scala"]),
-        plugins = [
-            kind_projector_plugin,
-        ],
         resources = glob(["src/main/resources/**/*"]),
         scala_deps = [
             "@maven//:com_github_scopt_scopt",
-            "@maven//:com_typesafe_akka_akka_actor",
-            "@maven//:com_typesafe_akka_akka_http_core",
-            "@maven//:com_typesafe_akka_akka_stream",
```

Comment:
These deps are no longer needed. Now akka is only linked via `//language-support/scala/bindings-akka` in `deps`.